### DISCUSSION TITLE
Playback Engine for OpenTimelineIO sequences

### DISCUSSION THREAD

**Discussion by maptz** [AcademySoftwareFoundation/OpenTimelineIO#863] [Q&A] [answered]
I understand that playback isn't the purpose of OpenTimelineIO, but is there a playback engine that would allow for playback using certain adapters?  Is there room for one?

**Comment by apetrynet** [comment, marked as answer]
Hi! 

I believe the latest gstreamer should be able to playback an `.otio` file with `ges-launch-1.0 timeline.otio`  
You could also try the [mlt](https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/799) adapter and playback using melt. 
And there's the [RV plugin](https://github.com/PixarAnimationStudios/OpenTimelineIO/tree/master/contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader)

**Reply by maptz** [reply]
That's interesting. Thanks @apetrynet . 

I'll dig into the gstreamer implementation a bit. Looks very interesting.

**Comment by jminor** [comment]
Recently OTIO support was released in these playback tools:
- Autodesk's RV: http://www.cgchannel.com/2021/12/autodesk-ships-rv-2022/
- ftrack's cineSyncPlay (including the free version): https://www.ftrack.com/en/cinesync
- tlRender: https://github.com/darbyjohnston/tlRender

A few others are listed here also: https://github.com/PixarAnimationStudios/OpenTimelineIO/wiki/Tools-and-Projects-Using-OpenTimelineIO
